web-extensions/chrome/engine/datamodel.js

```diff
@@ -435,17 +435,27 @@ BmcDataAPI.prototype.updateComic = function(comicId, chapter, page, cb) {
          * once. We may actually want to accept that, though.
          * Similarly, we may want to allow backwards-tracking too.. ?
          */
-        if (payload.tracking.chapter > chapter
-            || (payload.tracking.chapter == chapter
-                && (payload.tracking.page && page
-                    && payload.tracking.page > page))) {
-            return cb(new Error(LOCALIZATION.getString('S22')));
+        if (payload.tracking.chapter !== undefined && payload.tracking.page !== undefined) {
```

Comment:
Since the code was originally written, it's become possible to have an undefined `page` (cases where a chapter can be read as a whole and not page-by-page).
We should account for it.